extensions/applicationinsights-dependencies-js/Tests/Selenium/dependencies.tests.ts

```diff
@@ -1,6 +1,8 @@
-import { AjaxTests, AjaxPerfTests } from "./ajax.tests";
+import { AjaxTests, AjaxPerfTests, AjaxFrozenTests } from "./ajax.tests";
 
 export function runTests() {
     new AjaxTests().registerTests();
     new AjaxPerfTests().registerTests();
+    // The frozen tests are browser only as Phantom JS doesn't support enought of the spec to fail correctly
+    //new AjaxFrozenTests().registerTests();
```

Comment:
Are we meant to manually uncomment to test in browser? Is it possible to register these tests if we know we aren't in phantom?

Reply:
Yes, as this is a patch I tested but didn't want to spend the time building the additional framework (at this point) to automate this.